npm/webpack-dev-server/__snapshots__/makeWebpackConfig.spec.ts.js

```diff
@@ -29,10 +30,14 @@ exports['makeWebpackConfig ignores userland webpack `output.publicPath` and `dev
   },
   "devServer": {
     "progress": true,
-    "overlay": false
+    "overlay": true,
+    "client": {
```

Comment:
Snapshot from previous runs? Were not tweaking the overlay so I wouldn't expect this snapshot to change execpt for the `emitOnErrors`

Reply:
Yeah not sure what that was about... should be worked out now 👍🏻